python/rikai/io.py

```diff
@@ -164,6 +165,12 @@ def open_uri(
         # This is a local file
         return open(uri, mode=mode)
     elif parsed_uri.scheme in ("http", "https"):
+        if http_headers is None:
+            http_headers = {}
+        if "User-Agent" not in http_headers:
```

Comment:
Just curious why this is needed (or what other values we should expect)?

Reply:
Now wikipedia expects a http request has agent header.  Useful in the tests for now.

I leave this to customizable so that some of the downstream customers can customize it for access control?